### ISSUE TITLE
video pause when pay to half but audio is normal

### ISSUE THREAD

**Issue by dushicai** [google/ExoPlayer#5593]
When I play to a location where the video stream is damage , lead to video pause but audio is normal
(damage video track length about 2 seconds,MediaPlayer can play it normal and mosaic will appear)
How to fix this problem?

Media Info:
 Duration: 00:04:58.33, start: 1.466667, bitrate: 3080 kb/s
  Program 1 
    Metadata:
      service_name    : LIVE
      service_provider: KSYUN
    Stream #0:0[0x100]: Video: h264 (High) ([27][0][0][0] / 0x001B), yuv420p(progressive), 720x480 [SAR 32:27 DAR 16:9], 30 fps, 30 tbr, 90k tbn, 60 tbc
    Stream #0:1[0x101]: Audio: mp2 ([3][0][0][0] / 0x0003), 44100 Hz, stereo, fltp, 192 kb/s
    Stream #0:2[0x102]: Audio: mp2 ([3][0][0][0] / 0x0003), 44100 Hz, stereo, fltp, 192 kb/s

**Comment by ojw28** [comment]
We generally expect input to be well formed. What's the valid use case where you'd need to play corrupt content?

Please provide a test stream if you want us to take a closer look; there's not much we can do to help without one.

**Comment by dushicai** [comment]
file link:
https://space.dingtalk.com/s/gwHOAELdvgLOAHH5dgPaACBjZDNjZTkzYjcwNmE0YjhmYjYwMjBmYTQyNTRhOTQ3MA 

Verification Code: ZsMZ
 
The error happen time: 3m1s
hanks your help😁

**Comment by dushicai** [comment]
https://share.weiyun.com/5lVsNlc

error happen time :3m1s

thank you😁

**Comment by dushicai** [comment]
> We generally expect input to be well formed. What's the valid use case where you'd need to play corrupt content?
> 
> Please provide a test stream if you want us to take a closer look; there's not much we can do to help without one.

https://drive.google.com/file/d/1l0XuEfyrPsE-jg98kndVlpfzcqkKSmFH/view?usp=sharing
error happen time :3m1s
thanks

**Comment by ojw28** [comment]
On a Galaxy S8 the video plays through with some corruption, but does not freeze. On an emulator I saw the video freeze. The video freezes on VLC running on MAC as well. I think the outcome likely depends on the underlying video decoder in the platform. I don't think there's much value in spending time trying to make the failure mode less bad. As above; we generally expect input to be well formed.

Closing as bad media.
